azurerm_eventgrid_domain - adds minimum tls argument (#28952)
hashicorp/terraform-provider-azurerm: website/docs/r/eventgrid_domain.html.markdown

* `input_mapping_default_values` - (Optional) A `input_mapping_default_values` block as defined below. Changing this forces a new resource to be created.

> * `min_tls_version` - (Optional) The minimum supported TLS version for the EventGrid Domain. Possible values are `1.0`, `1.1`, and `1.2`. Defaults to `1.2` for new Domains.

Reviewer: ```suggestion
* `min_tls_version` - (Optional) The minimum supported TLS version for the EventGrid Domain. Possible value is `1.2`. Defaults to `1.2` for new Domains.
```
